tests/functests/instancetype_test.go

```diff
@@ -231,11 +231,16 @@ var _ = Describe("Common instance types func tests", func() {
 				map[string]string{"amd64": "opensuse.leap", "arm64": "opensuse.leap"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("opensuse")}),
 			Entry("[test_id:TODO] Oracle Linux 8", ol8ContainerDisk, "u1.2xmedium",
-				map[string]string{"amd64": "oraclelinux", "arm64": "oraclelinux"},
+				map[string]string{"amd64": "oraclelinux"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("cloud-user")}),
+			Entry("[test_id:TODO] Oracle Linux 8", ol8ContainerDisk, "u1.2xmedium",
+				map[string]string{"arm64": "oraclelinux"},
+				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("opc")}),
 			Entry("[test_id:TODO] Oracle Linux 9", ol9ContainerDisk, "u1.2xmedium",
-				map[string]string{"amd64": "oraclelinux", "arm64": "oraclelinux"},
+				map[string]string{"amd64": "oraclelinux"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("cloud-user")}),
+			Entry("[test_id:TODO] Oracle Linux 9", ol9ContainerDisk, "u1.2xmedium",
+				map[string]string{"arm64": "oraclelinux"},
```

Comment:
you missed `[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("opc")}),`

Reply:
:facepalm:
Adjusted